app/logic/serviceLearningCourses.py

```diff
@@ -189,7 +189,12 @@ def withdrawProposal(courseID) -> None:
     except DoesNotExist:
         print(f"File, {AttachmentUpload.fileName}, does not exist.")
 
-    # delete course object
+    # deletes course
```

Comment:
This comment is not needed, the line right after describes what is going on pretty well in my opinion